ci-operator/jobs/openshift/lightspeed-service/openshift-lightspeed-service-main-presubmits.yaml

```diff
@@ -0,0 +1,137 @@
+presubmits:
+  openshift/lightspeed-service:
+  - agent: kubernetes
+    always_run: true
```

Comment:
let's make this false for now, there's no reason to run an e2e job that's going to spin up a full cluster just for us to do nothing with it.  We can change it when we get to the point of actually wanting to deploy something in a cluster or otherwise need a cluster for something.

Reply:
Yes sounds good to me. Changed as suggested.